tools/topology/sof-sdw-rt711.m4

```diff
@@ -0,0 +1,172 @@
+#
```

Comment:
No, that's wrong to add yet another file just for a limited test configuration.

@libinyang you should start with the configuration we had for the Chromebook and remove the amps with a macro.

We should really have TWO baselines (all SoundWire, or mixed SoundWire/dmic).

Reply:
@plbossart Do you mean reuse sof-tgl-rt711-rt1308 and add DMIC support in it?

Reply:
@libinyang this topology already has DMIC, so all you need to do is remove the RT1308 amp support with a macro

Reply:
> @libinyang this topology already has DMIC, so all you need to do is remove the RT1308 amp support with a macro

@plbossart Got it. I will do it.